webapp/src/cloud.ts

```diff
@@ -175,15 +175,18 @@ function resetAsync(): Promise<void> {
     return Promise.resolve();
 }
 
-let inProgressSyncPromise: Promise<any> = Promise.resolve()
+let inProgressSyncPromise: Promise<pxt.workspace.Header[]>;
```

Comment:
If you await a null or undefined promise, is that okay?

Reply:
awaiting null / a non promise is fine, implementation wise it either is a no-op or just wraps in Promise.resolve(..)

Reply:
Awesome thanks!